go.mod

```diff
@@ -1,12 +1,12 @@
-module github.com/plgd-dev/go-coap/v2
+module github.com/plgd-dev/go-coap/v3
 
 go 1.18
 
 require (
 	github.com/dsnet/golib/memfile v1.0.0
-	github.com/pion/dtls/v2 v2.1.5
-	github.com/plgd-dev/kit/v2 v2.0.0-20211006190727-057b33161b90
-	github.com/stretchr/testify v1.8.0
+	github.com/pion/dtls/v2 v2.1.6-0.20220815090156-a04cfccb23e5
+	github.com/pion/udp v0.1.1
+	github.com/stretchr/testify v1.7.1
```

Comment:
This seems weird to me, why is the version here (v1.7.1) older than the previous one (v1.8.0)?

Reply:
fixed